docs/RC/OnInteriorVehicleData/index.md

```diff
@@ -13,6 +13,8 @@ Purpose
 
 The HMI must send RC.OnInteriorVehicleData notification to SDL when module settings were changed in case of button press event or after SetInteriorVehicledata request was processed.
 
+The HMI must always include a `moduleId` for a `ModuleData` in OnInteriorVehicleData notification.
```

Comment:
Maybe put this point inside a 
```markdown
!!! MUST   

!!!
``` 
block

Reply:
updated in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/171/commits/59936e40b204b542b7de0a6b4a5706e0ccec14f8